Missing Iterator Utils in apoc .jar file

I've ran into this issue both with apoc 3.3.0.2/neo4j 3.3.3 as well as with apoc 3.2.3.6/neo4j 3.2.6 both running in docker containers.

When calling: 
```
match (n:pharos_drug) where n.name="ibuprofen" call apoc.path.subgraphAll(n, {maxLevel:1, relationshipFilter:'targets>'}) yield nodes, relationships return nodes, relationships
``` 
I received the following error: "Failed to invoke procedure `apoc.path.subgraphAll`: Caused by: java.lang.NoClassDefFoundError: org/apache/commons/collections/IteratorUtils"

After reading this error I thought that maybe I missed a step when following the [installation directions](https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_using_apoc_with_neo4j_docker_image) in the documentation but after combing through the documentation for a while I couldn't find any extra configuration steps for installation other than those for [hive](https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_support_for_hive_with_kerberos_auth), [mongoDB](https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_interacting_with_mongodb), and [couchbase.](https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_interacting_with_couchbase)

To fix the issue I unpacked the apoc .jar file and put [apache commons collections 3.2.2](https://commons.apache.org/proper/commons-collections/download_collections.cgi) into the directory org/apache/commons/collections/ and then repacked it. Then, after putting this new .jar file into the neo4j plugins directory everything worked fine.

This leads me to believe that there may be missing dependencies (apache commons collections) in the .jar files (at least for 3.3.0.2 & 3.2.3.6) available for download [here.](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases)

I am new to both neo4j and apoc so I apologize ahead of time if there is anything I missed in the documentation or if there is a config file I failed to use or if there is some other obvious thing I missed when looking into this issue.

--- a/src/main/java/apoc/path/RelationshipSequenceExpander.java
+++ b/src/main/java/apoc/path/RelationshipSequenceExpander.java
@@ -1,14 +1,12 @@
 package apoc.path;
 
-import org.apache.commons.collections.IteratorUtils;
 import org.neo4j.graphdb.*;
-import org.neo4j.graphdb.impl.StandardExpander;
 import org.neo4j.graphdb.traversal.BranchState;
+import org.neo4j.helpers.collection.Iterators;
 import org.neo4j.helpers.collection.NestingIterator;
 import org.neo4j.helpers.collection.Pair;
 
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Iterator;
 import java.util.List;
 
@@ -84,7 +82,7 @@ public class RelationshipSequenceExpander implements PathExpander {
             stepRels = relSequences.get((initialRels == null ? depth : depth - 1) % relSequences.size());
         }
 
-        return IteratorUtils.toList(
+        return Iterators.asList(
          new NestingIterator<Relationship, Pair<RelationshipType, Direction>>(
                 stepRels.iterator() )
         {

--- a/src/main/java/apoc/schema/Schemas.java
+++ b/src/main/java/apoc/schema/Schemas.java
@@ -3,7 +3,6 @@ package apoc.schema;
 import apoc.result.AssertSchemaResult;
 import apoc.result.ConstraintRelationshipInfo;
 import apoc.result.IndexConstraintNodeInfo;
-import org.apache.commons.collections.CollectionUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.neo4j.graphdb.GraphDatabaseService;
 import org.neo4j.graphdb.Label;
@@ -156,10 +155,6 @@ public class Schemas {
 
         db.execute(String.format("CREATE INDEX ON :`%s` (%s)", label, String.join(",", backTickedKeys)));
         return new AssertSchemaResult(label, keys).created();
-    }
-
-    private Boolean compareKeys(List<String> keys1, List<Object> keys2) {
-        return CollectionUtils.isEqualCollection(keys1, keys2);
     }
 
     private Map<String, List<Object>> copyMapOfObjects(Map<String, List<Object>> input) {

